.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
```

Comment:
I honestly don't think we should be making that choice on behalf of people. Sometimes people might just need a quick nudge in the right direction, and requiring them to sign up for Discourse (Gitter can be used with a github account, so it's practically frictionless), which IIRC isn't immediate as it requires the account to be approved first, and create a new post, and wait until someone responds, is putting up way more barriers than I'd be comfortable agreeing with, only for the benefit of potentially finding that question down the road later on -- **if** that's at all useful, which may not always be the case.

I'd much rather have people either suggest the asker in Gitter to post in Discourse if their issue is common and/or complex, or using the opportunity to add caveats to the manual or to a FAQ somewhere.

By the way, quoting @StefanKarpinski [about moving the google groups contents to Discourse](https://discourse.julialang.org/t/import-google-groups-emails/453/5):

> Personally, I think the clean break is better. We can still link to posts on Google Groups and any content there that proves particularly valuable to have around can be moved over in a more curated fashion. Volume is the enemy of effective search.